pipeline/splicing/references.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# Download and verify references - to be called from within the docker
+echo "Downloading references. This may take 1-2 hours..."
+wget -N -P /references http://hgdownload.cse.ucsc.edu/goldenPath/hg38/bigZips/hg38.fa.gz
```

Comment:
we also host this file at http://brcaexchange.org/backend/downloads/resources/hg38.fa.gz, not sure if it matters where we get it